### ISSUE TITLE
Q3 roadmap

### ISSUE THREAD

**Issue by bartlomieju** [denoland/vscode_deno#879]
This is the roadmap for Q3 of 2023.

Over the past few days we've triaged every single open issue in this repository. Over 15% of issues have been closed as already fixed, but we have identified around 20 high priority issues that severely impact experience of using Deno in your favorite code editor.

Below is the list of the high priority issues that we intend to address in Q3 2023:

Active work:

- [ ] https://github.com/denoland/vscode_deno/issues/837
- [ ] https://github.com/denoland/vscode_deno/issues/789

Needs more investigation:

- [ ] https://github.com/denoland/vscode_deno/issues/878
- [ ] https://github.com/denoland/vscode_deno/issues/555
- [ ] https://github.com/denoland/vscode_deno/issues/616

Waiting on workspace support:
- [ ] https://github.com/denoland/vscode_deno/issues/488
- [ ] https://github.com/denoland/vscode_deno/issues/787

Complete:

- [x] https://github.com/denoland/vscode_deno/issues/743
- [x] https://github.com/denoland/vscode_deno/issues/784
- [x] https://github.com/denoland/vscode_deno/issues/825
- [x] https://github.com/denoland/vscode_deno/issues/835
- [x] https://github.com/denoland/vscode_deno/issues/860
- [x] https://github.com/denoland/vscode_deno/issues/410
- [x] https://github.com/denoland/vscode_deno/issues/867
- [x] https://github.com/denoland/vscode_deno/issues/841 - Addressed via new diagnostic which adds instructions on what to do.
- [x] https://github.com/denoland/vscode_deno/issues/850 - It might not be possible for us to add breadcumbs here due to a limitation in vscode
- [x] https://github.com/denoland/vscode_deno/issues/856

**Comment by deckchairlabs** [comment]
Is #875 a duplicate of something else?

**Comment by evelant** [comment]
I think #878 #835 and #581 may all be the same root issue

**Comment by bartlomieju** [comment]
> I think #878 #835 and #581 may all be the same root issue

That's correct. We will fix them once we add native support for workspaces in Deno.

**Comment by bartlomieju** [comment]
Q3 is over. Closing this issue and I'll open a new one for Q4.
